### PR TITLE
[A11y] Make speed plugin keyboard + SR friendly

### PR DESCRIPTION
**Pull request by tennety**
- adds keyboard and SR navigation to speed selector
- makes menu hover timeout configurable

### CODE REVIEW

File: src/js/mep-player.js [hunk 1]
```diff
@@ -94,6 +94,8 @@ function isAfter(sourceNode, targetNode) {
 		controlsTimeoutMouseEnter: 2500,
 		// Time in ms to trigger the timer when mouse leaves
 		controlsTimeoutMouseLeave: 1000,
+    // Time in ms to hide menu on when mouse leaves
+    menuTimeoutMouseLeave: 500,
```
Comment (mziwisky) [inline src/js/mep-player.js]: formatting fail -- looks like the indentation of the surrounding source is two spaces followed by two tabs, _clearly_ a sane and elegant standard, and one that i personally have configured my editor to use by default henceforth.

File: src/js/mep-feature-speed.js [hunk 2]
```diff
@@ -113,14 +121,93 @@
...
 					});
 			}
+		},
+
+		hideSpeedSelector: function () {
+			this.speedButton.find('.mejs-speed-selector')
+				.addClass('mejs-offscreen')
+				.attr('aria-expanded', 'false')
+				.attr('aria-hidden', 'true')
+				.find('input[type=radio]') // make radios not fucusable
```
Comment (mziwisky) [inline src/js/mep-feature-speed.js]: fuc who-able? ;)

File: src/js/mep-feature-speed.js [hunk 3]
```diff
@@ -113,14 +121,93 @@
...
 								speedButton.find('.mejs-speed-translations').outerHeight(true))
 							.css('top', (-1 * speedSelector.height()) + 'px');
+						})
+
+						// hover
+						.hover(function() {
+							clearTimeout(hoverTimeout);
+							player.showSpeedSelector();
+						}, function() {
+							var self = $(this);
```
Comment (mziwisky) [inline src/js/mep-feature-speed.js]: this `self` could be removed

File: src/js/mep-feature-sourcechooser.js [hunk 4]
```diff
@@ -31,7 +31,7 @@
 						var self = $(this);
```
Comment (mziwisky) [inline src/js/mep-feature-sourcechooser.js]: i know this isn't your change, but this `self` could be removed

**Review by mziwisky — CHANGES_REQUESTED** [review]
just a few cleanups, nothing substantial

File: src/js/mep-feature-speed.js (hunk 2 above)
Comment (tennety) [reply src/js/mep-feature-speed.js]: [facepalm]

**Review by mziwisky — APPROVED** [review]

**Pull request merged**